### ISSUE TITLE
missing Animated.diffClamp

### ISSUE THREAD

**Issue by ntharim** [animatedjs/animated#57]
https://facebook.github.io/react-native/docs/animated.html#diffclamp

https://github.com/facebook/react-native/blob/master/Libraries/Animated/src/AnimatedImplementation.js

**Comment by browniefed** [comment]
Yep, this is an older copy of Animated. If you'd like to add it I'd happily accept a PR

**Comment by browniefed** [comment]
https://github.com/animatedjs/animated/pull/64
